cylc/flow/scripts/lint.py

```diff
@@ -443,6 +443,7 @@ def list_wrapper(line: str, check: Callable) -> Optional[Dict[str, str]]:
     'S': 'Style'
 }
 LINE_LEN_NO = 'S012'
+EMPTY_FILE = 'S000'   # Reference for a completely empty file.
```

Comment:
Do we need a code for this?

Can't we just exit `0` (nothing in the file to complain about).

Reply:
Yes. Fair point.